packages/hydrooj/src/handler/manage.ts

```
@@ -184,27 +183,26 @@ class SystemConfigHandler extends SystemHandler {
         } catch (e) { }
         this.response.body = {
-            schema: Schema.intersect(SystemSettings).toJSON(),
+            schema: Schema.intersect(this.ctx.config.settings).toJSON(),
             value,
         };
     }
 
     @requireSudo
     @param('value', Types.String)
-    async post(domainId: string, value: string) {
+    async post({ }, value: string) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)